core/src/test/resources/health-wait-strategy-dockerfile/write_file_and_loop.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/ash
 
+set -e
```

Comment:
Perhaps we should do:
```suggestion
set -ex
```
so that the shell automatically logs the command that's about to be run.

We could actually just get rid of all the `echo`s then...

Reply:
Good idea, added your suggestion accordingly.